PowerEditor/src/ScitillaComponent/FindReplaceDlg.rc

```diff
@@ -37,6 +37,7 @@ EXSTYLE WS_EX_TOOLWINDOW
 CAPTION "Replace"
 FONT 8, "MS Shell Dlg", 0, 0, 0x0
 BEGIN
+    AUTOCHECKBOX    "&Quick find", ID_QUICK_FIND, 129, 115, 125, 8, 0, WS_EX_LEFT

```

Comment:
This line was generated by ResEdit and I didn't looked what is the difference between using syntax with CONTROL, I guess I should use it as well for consistency.